test/persistence-hooks.suite.js

```diff
@@ -137,7 +137,7 @@ module.exports = function(dataSource, should) {
         });
       });
 
-      it('applies updates from `before save` hook', function(done) {
+      it('applies updates from `before save` hook for create', function(done) {
```

Comment:
In my opinion, the test suite name "PersistedModel.create" already provides the fact that these tests are testing "create" operation, therefore I think the suffix " for created" is redundant. Could you please remove?